### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by tkusano**
Updated

### CODE REVIEW

**Comment by tkusano** [comment]
@dolciss @Hima-Zinn @middlingphys @hibiki
Updated translation, could you review this?

**Review by dolciss — COMMENTED** [review]
Thanks for the mentions.
LGTM👍

**Review by Hima-Zinn — COMMENTED** [review]
It looks fine.

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -1905,6 +1939,10 @@ msgstr "有効"
 msgid "End of feed"
 msgstr "フィードの終わり"
 
+#: src/tours/Tooltip.tsx:159
+msgid "End of onboarding tour window. Do not move forward. Instead, go backward for more options, or press to skip."
+msgstr "オンボーディングツアー・ウインドウ終了。前へ移動しないでください。代わりに、戻って他のオプションを見るか、スキップしてください。"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:1944]:
```suggestion
msgstr "オンボーディングツアー・ウインドウ終了。先へ進まないでください。代わりに、戻って他のオプションを見るか、スキップしてください。"
```

File: src/locale/locales/ja/messages.po [hunk 2]
```diff
@@ -5390,6 +5466,10 @@ msgstr "{displayName}とのチャットを開始"
 msgid "Start chatting"
 msgstr "チャットを開始"
 
+#: src/tours/Tooltip.tsx:99
+msgid "Start of onboarding tour window. Do not move backward. Instead, go forward for more options, or press to skip."
+msgstr "オンボーディングツアー・ウインドウ開始。前へ移動しないでください。代わりに、戻って他のオプションを見るか、スキップしてください。"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5471]:
```suggestion
msgstr "オンボーディングツアー・ウインドウ開始。前へ戻らないでください。代わりに、進んで他のオプションを見るか、スキップしてください。"
```

File: src/locale/locales/ja/messages.po (hunk 1 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: fixed!

File: src/locale/locales/ja/messages.po (hunk 2 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: fixed!!

**Review by pfrazee — APPROVED** [review]
Thanks everyone!!

**Pull request merged**